src/game/state.py

```diff
@@ -66,13 +67,13 @@ def check_position_availability(self, x, y):
     def switch_player(self):
         self.game_turn.player_one_active = not self.game_turn.player_one_active
 
-    def check_winner(self):
+    def check_winner(self, players):
```

Comment:
"check_winner" is not a good name for this function. Moreover, 1 function = 1 task. Here, the function is responsible of "message" change and return a boolean

I would create a function to get the current winner (or nul if no one) and another to assign the message from the returned value instead.